pagopa/pn-frontend
# PN-11969 - Remove context from courtesy contacts

## Short description
Remove context from courtesy contacts

## List of changes proposed in this pull request
- Moved logic from context to CourtesyContactItem
- Updated tests

## How to test
Login with pf/pg -> go to recapiti page -> add/remove/edit a phone and email contact and check that everything is ok
